deployments/helm-chart/templates/controller-appprotect-dos-arb.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.controller.appprotectdos.enable }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "nginx-ingress.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+    spec:
+      containers:
+        - name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+          image: "{{ .Values.controller.appprotectdos.arbitrator.repository }}"
```

Comment:
do we need to expose any readiness and liveliness probes for the arbitrator?

Reply:
No need to expose any probes for the arbitrator